### DISCUSSION TITLE
Hearing All Players (Dead or Alive)

### DISCUSSION THREAD

**Discussion by Dexteriority** [winstxnhdw/lc-hax#327] [Ideas]
Basically the idea is to create a /command that can toggle between normal hearing and all hearing.
Like you can hear the spectators, you can hear people inside the dungeon even if you are in the ship vice versa.
It will be awesome! :3

**Comment by winstxnhdw** [comment]
Oh, joep wanted this feature too, but I didn't know about toggling between normal and all hearing. That's a great idea. I'll look into it.

**Reply by Dexteriority** [reply]
Thanks! We count on you!

**Reply by winstxnhdw** [reply]
I need a good command name for this.

**Reply by winstxnhdw** [reply]
`/eavesdrop` maybe?

**Reply by Dexteriority** [reply]
I mean the most simple command can be is /hear but /eavesdrop is good choice😂😂😂

**Reply by winstxnhdw** [reply]
`/hear` it is I guess

**Reply by Dexteriority** [reply]
Up to you brother! All good!

**Reply by winstxnhdw** [reply]
Tested and added in latest.

**Reply by Dexteriority** [reply]
Great work as always! We know we can count on you🙌
